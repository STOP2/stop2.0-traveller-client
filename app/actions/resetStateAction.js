export const RESET_STATE = 'RESET_STATE'

export let resetState = function()
{
    return {type: RESET_STATE}
}
