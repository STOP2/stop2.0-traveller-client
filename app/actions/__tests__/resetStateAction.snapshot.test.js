import { resetState } from '../resetStateAction';

describe('reset state action', () => {
  it('creates a RESET_STATE action', () => {
    expect(resetState()).toMatchSnapshot();
  });
});
