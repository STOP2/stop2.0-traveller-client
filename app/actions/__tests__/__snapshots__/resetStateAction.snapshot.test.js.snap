exports[`reset state action creates a RESET_STATE action 1`] = `
Object {
  "type": "RESET_STATE",
}
`;
